rock-n-rollup/src/core/service.rs

```diff
@@ -269,7 +270,8 @@ where
 
 #[cfg(test)]
 mod tests {
-    use crate::core::{runtime::MockRuntime, Application, Runtime};
+    use crate::core::{runtime::MockRuntime, runtime::Runtime, Application};
+    //use tezos_smart_rollup_host::runtime::Runtime;
```

Comment:
this comment can be removed

Reply:
it has been removed at this [commit](3d56b448ce27ff2ee25b9e46e3b86aecbbb74896)